code/ATMOSPHERICS/components/binary_devices/circulator.dm

```diff
@@ -15,6 +15,9 @@
 	anchored = 1
 	density = 1
 
+	var/global/const/CIRC_COLD = 1

```

Comment:
we've had this argument before and I really don't wanna have it again
both are used throughout the codebase and obviously there's no maintainer consensus on it so I'm just gonna leave it as it is